include/pfs/drp/stella/Spectra.h

```diff
@@ -3,28 +3,13 @@
 
 #include <vector>
 #include <iostream>
-#include <cmath>
-//#pragma clang diagnostic push
-//#pragma clang diagnostic ignored "-Wunused-variable"
-//#pragma clang diagnostic pop
-#include "boost/algorithm/string/trim.hpp"
-#include "lsst/base.h"
-#include "lsst/afw/geom.h"
-#include "lsst/afw/image/MaskedImage.h"
```

Comment:
Added the include `lsst/afw/image/Image.h` which was only in a file included by `Spectra.h`